src/submission/submission-entities.ts

```diff
@@ -372,8 +372,7 @@ export const BatchNameRegex: Record<ClinicalEntitySchemaNames, RegExp[]> = {
   [ClinicalEntitySchemaNames.HORMONE_THERAPY]: [/^hormone_therapy.*\.tsv$/],
 };
 
-// assumption: one field uniquely identifies a clinical type record in a batch of records
-export const ClinicalUniqueIndentifier: {
+export const ClinicalUniqueIdentifier: {
```

Comment:
not an indentifier...

Reply:
should we keep one of those Unique identifiers constant or both are necessary ?

Reply:
I'm not sure I understand?

Reply:
@blabadi ok I have removed the other identifier, there is only one now :+1: